tests/index_test.js

```diff
@@ -204,6 +204,8 @@ describe('StyleSheet.create', () => {
     it('works for empty stylesheets and styles', () => {
         const emptySheet = StyleSheet.create({});
 
+        assert.ok(emptySheet);
```

Comment:
I added this because otherwise ESLint reports `emptySheet` as being an unused variable.